### DISCUSSION TITLE
Is there any way to add Attachments on listitem?

### DISCUSSION THREAD

**Discussion by gopaldahal** [pnp/pnpcore#491] [Q&A] [answered]
Didn't find list item Attachments operations.

https://s-kainet.github.io/sp-rest-explorer/#/entity/SP.ListItem

![image](https://user-images.githubusercontent.com/27617624/122343597-97c74e80-cf65-11eb-9d70-f6abb33538b4.png)

**Comment by jansenbe** [comment]
@gopaldahal : we currently don't support that yet. If you're blocked in your development due to missing this feature, then please create an issue for it so that we can track it.

**Comment by gopaldahal** [comment, marked as answer]
This is now supported by @jansenbe. Thank you.
Looking into it and will post if anything needs to be rectified

Sharing is Caring!!
